### PR TITLE
Add Annotation helpers

### LINKED ISSUE #40: Add Annotation
## Description

This helper will add an existing Annotation to the object's `annotations` key

## Function name / Parameters / Return

Function name: `.add_annotation(annotation)`

Parameters: `annotation` - the Prezi3 `Annotation` object to be added

Returns: `None`

## Objects

- [ ] Base Object (all Prezi3 Classes)
- [x] Collection
- [x] Manifest
- [x] Canvas
- [x] Range
- [ ] AnnotationCollection
- [ ] AnnotationPage
- [ ] Annotation
- [x] Content Resource - Image
- [x] Content Resource - Video
- [x] Content Resource - Text
- [ ] Service
- [ ] Provider
- [ ] Other - please list below

## Considerations

Annotations must exist within an `AnnotationPage` object, so the helper must create and add one if one is not already present. Otherwise the Annotation should be added to the first existing `AnnotationPage`

### PR DESCRIPTION
**Pull request by robcast**
Adding Annotation helper methods.

Fixes #40 and #60

### CODE REVIEW

**Comment by azaroth42** [comment]
Fixed import conflict so tests would run.

File: iiif_prezi3/helpers/annotation_helpers.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+                        Manifest, Range)
+
+
+class AnnotationHelpers:
+
+    def add_annotation(self, annotation):
+        """Adds the annotation object to the (AnnotationPage object in the) annotations property.
+
+        Args:
+          annotation (iiif-prezi3.skeleton.Annotation): the Annotation to add
```
Comment (digitaldogsbody) [inline iiif_prezi3/helpers/annotation_helpers.py:12]: This should be `iiif_prezi3` as that's the module's name (vs the project which is with the -)

Also it might be better to skip the skeleton part if it's possible, as we need users to import using the chained import triggered by the main module's `__init__.py` so abstracting away the detail will lessen the chance of an accidental error if users follow the autogenerated docs.

tl;dr: I think this should be `annotation (iiif_prezi3.Annotation)` but with the caveat that I don't know how this will work with the auto doc generator.

**Review by digitaldogsbody — APPROVED** [review]
Only potential change is with the docstring, otherwise this looks great :)

**Review by giacomomarchioro — APPROVED** [review]

File: iiif_prezi3/helpers/annotation_helpers.py (hunk 1 above)
Comment (robcast) [reply iiif_prezi3/helpers/annotation_helpers.py:12]: Good point. (I blindly copy-pasted from canvas_helpers.py ;-)
